performance-tests/bench_2/test_controller/main.cpp

```diff
@@ -528,53 +629,95 @@ int handle_reports(const Bench::NodeController::ReportSeq& nc_reports,
   result_out << "  Max Start Time: " << max_start_time << " seconds" << std::endl;
   result_out << "  Max Stop Time: " << max_stop_time << " seconds" << std::endl;
   result_out << "  Max Destruction Time: " << max_destruction_time << " seconds" << std::endl;
-
   result_out << std::endl;
 
   result_out << "Discovery Stats:" << std::endl;
-  result_out <<
-    (total_undermatched_readers != 0 ? "  ERROR: " : "  ") <<
+  result_out << (total_undermatched_readers != 0 ? "  ERROR: " : "  ") <<
     "Total Undermatched Readers: " << total_undermatched_readers <<
     (total_undermatched_writers != 0 ? ", ERROR: " : ", ") <<
     "Total Undermatched Writers: " << total_undermatched_writers << std::endl;
   //result_out << "  Max Discovery Time Delta: " << max_discovery_time_delta << " seconds" << std::endl;
-
   result_out << std::endl;
 
   consolidated_discovery_delta_stats.pretty_print(result_out, "discovery time delta");
-
   result_out << std::endl;
 
   result_out << "DDS Sample Count Stats:" << std::endl;
   result_out << "  Total Lost Samples: " << total_lost_sample_count << std::endl;
   result_out << "  Total Rejected Samples: " << total_rejected_sample_count << std::endl;
-
   result_out << std::endl;
 
   result_out << "Data Count Stats:" << std::endl;
   result_out << "  Total Out-Of-Order Data Samples: " << total_out_of_order_data_count << std::endl;
   result_out << "  Total Duplicate Data Samples: " << total_duplicate_data_count << std::endl;
   result_out << "  Total Missing Data Samples: " << total_missing_data_count << std::endl;
-
   result_out << std::endl;
 
   result_out << "Data Timing Stats:" << std::endl;
-
   result_out << std::endl;
 
-  consolidated_latency_stats.pretty_print(result_out, "latency");
-
+  consolidated_latency_stats.pretty_print(result_out, "latency", "  ", 1);
```

Comment:
Any particular reason these got indented but not the ones above?

Reply:
It looks to me that the stats info for latency, jitter, round-trip latency, round-trip jitter all fall under the same "Data Timing Stats:" section. Without an additional indentation level for them, it looks like "Data Timing Stats:" is its own section without any data.